firebase_admin/tenant_mgt.py

```diff
@@ -91,7 +92,8 @@ def get_tenant(tenant_id, app=None):
 
 
 def create_tenant(
-        display_name, allow_password_sign_up=None, enable_email_link_sign_in=None, app=None):
+        display_name, allow_password_sign_up=None, enable_email_link_sign_in=None,
+        mfa_config=None, app=None):
```

Comment:
is this a breaking change for existing customer?

Reply:
No, since mfa_config is an optional field.